project.Makefile

```diff
@@ -117,54 +103,18 @@ local/usage_template.tsv: nmdc_schema/nmdc_materialized_patterns.yaml # replaces
 		--log-file $@.log.txt \
 		--report-style exhaustive
 
-examples/output/Biosample-exhaustive_report.yaml: src/data/problem/valid/Biosample-exhasutive.yaml # replaces misspelled Biosample-exhasutive_report target
-	$(RUN) exhaustion-check \
-		--class-name Biosample \
-		--instance-yaml-file $< \
-		--output-yaml-file $@ \
-		--schema-path src/schema/nmdc.yaml
+#examples/output/Biosample-exhaustive_report.yaml: src/data/valid/Biosample-exhasutive.yaml
+#	$(RUN) exhaustion-check \
+#		--class-name Biosample \
+#		--instance-yaml-file $< \
+#		--output-yaml-file $@ \
+#		--schema-path src/schema/nmdc.yaml
```

Comment:
I'd prefer this be removed instead of commented out, but I don't consider it to be a deal-breaker on a "backmerge," given all the moving parts. It can be removed in a subsequent PR.